SU2_CFD/src/solvers/CAdjEulerSolver.cpp

```diff
@@ -1183,311 +898,6 @@ void CAdjEulerSolver::SetForceProj_Vector(CGeometry *geometry, CSolver **solver_
-      }
-      nPointNearField++;
-    }
-
-    /*--- Note tha the first row is the azimuthal angle ---*/
-
-    nPointNearField = nPointNearField - 1;
-
-    for (AngleInt = 0; AngleInt < 180; AngleInt++)
-      IndexNF_inv[AngleInt] = -1;
-
-  if (IndexNF.size() <= 180) {
-    for (iIndex = 0; iIndex < IndexNF.size(); iIndex++)
-      IndexNF_inv[IndexNF[iIndex]] = iIndex;
-  }
-  else {
-    SU2_MPI::Error("", CURRENT_FUNCTION);
-  }
-
-  }
-
-  /*--- Compute the jump on the adjoint variables for the upper and the lower side ---*/
-
-  for (iMarker = 0; iMarker < nMarker; iMarker++)
-
-    if (config->GetMarker_All_KindBC(iMarker) == NEARFIELD_BOUNDARY)
-
-      for (iVertex = 0; iVertex < geometry->nVertex[iMarker]; iVertex++) {
-
-        iPoint = geometry->vertex[iMarker][iVertex]->GetNode();
-        Normal = geometry->vertex[iMarker][iVertex]->GetNormal();
-
-        Area = GeometryToolbox::Norm(nDim, Normal);
-
-        for (iDim = 0; iDim < nDim; iDim++)
-          UnitNormal[iDim] = Normal[iDim]/Area;
-
-        if (geometry->nodes->GetDomain(iPoint)) {
-
-          coord = geometry->nodes->GetCoord(iPoint);
-          DerivativeOF = 0.0;
-
-          /*--- Just in case the functional depend also on the surface pressure ---*/
-
-          WeightSB = 1.0-config->GetWeightCd();
-
-          su2double AoA, XcoordRot = 0.0, YcoordRot = 0.0, ZcoordRot = 0.0;
-
-          if (nDim == 2) XcoordRot = coord[0];
-          if (nDim == 3) {
-
-            /*--- Rotate the nearfield cylinder  ---*/
-
-            AoA = -(config->GetAoA()*PI_NUMBER/180.0);
-            XcoordRot = coord[0]*cos(AoA) - coord[2]*sin(AoA);
-            YcoordRot = coord[1];
-            ZcoordRot = coord[0]*sin(AoA) + coord[2]*cos(AoA);
-          }
-
-          switch (config->GetKind_ObjFunc()) {
-            case EQUIVALENT_AREA :
-
-              if (nDim == 2) AngleInt = 0;
-
-              if (nDim == 3) {
-
-                /*--- Compute the azimuthal angle of the iPoint ---*/
-
-                AngleDouble = fabs(atan(-YcoordRot/ZcoordRot)*180.0/PI_NUMBER);
-
-                /*--- Fix an azimuthal line due to misalignments of the near-field ---*/
-
-                su2double FixAzimuthalLine = config->GetFixAzimuthalLine();
-
-                if ((AngleDouble >= FixAzimuthalLine - 0.1) && (AngleDouble <= FixAzimuthalLine + 0.1)) AngleDouble = FixAzimuthalLine - 0.1;
-
-                AngleInt = SU2_TYPE::Short(floor(AngleDouble + 0.5));
-                if (AngleInt < 0) AngleInt = 180 + AngleInt;
-
-              }
-
-              if (AngleInt <= 60) {
-                iColumn = IndexNF_inv[AngleInt];
-
-                /*--- An azimuthal angle is not defined... this happens with MG levels ---*/
-
-                if (iColumn < 0.0) {
-                  if (IndexNF_inv[AngleInt+1] > 0) { iColumn = IndexNF_inv[AngleInt+1]; goto end; }
-                  if (IndexNF_inv[AngleInt-1] > 0) { iColumn = IndexNF_inv[AngleInt-1]; goto end; }
```

Comment:
I was almost surprised :smile:

Reply:
lol, that code looks like someone really hated `else if`